types/ndn-js/delegation-set.d.ts

```diff
@@ -0,0 +1 @@
+export type DelegationSet = any;
```

Comment:
This seems incorrect. https://github.com/named-data/ndn-js/blob/master/js/delegation-set.js#L51 shows its more like a class. Definitely shouldn't be any

Reply:
Many classes in ndn-js are rarely used so I declared them as `any`.
However, `DelegationSet` isn't one of them, so it now has full declaration, and merged into `interest.d.ts`.

Reply:
If the classes are not used you should still declare them as classes without declaring properties and methods in them for better shape than using any